### DISCUSSION TITLE
无法完成钓鱼

### DISCUSSION THREAD

**Discussion by SwingBin** [babalae/better-genshin-impact#459] [General]
会抛竿，会换饵料，就是不会自动追踪钓鱼条，不会起竿
![image](https://github.com/babalae/better-genshin-impact/assets/38577078/10c126b3-1839-4bb5-a2e2-d0902f2f066b)

**Comment by huiyadanli** [comment]
分辨率调整为 1080P ，多换几个点位尝试下

**Comment by huiyadanli** [comment]
参考 https://github.com/babalae/better-genshin-impact/issues/435
